tornado-api/src/main/java/uk/ac/manchester/tornado/api/DataRange.java

```diff
@@ -29,23 +29,18 @@
 public class DataRange {
 
     private final TornadoNativeArray tornadoNativeArray;
+    private final long totalSizeInBytes;
+    private final int elementSize;
```

Comment:
Out of curiosity, if we all use the same IntelliJ templates for code formatting, why is gets shuffled every time?